shared_memory_support.md

```diff
@@ -0,0 +1,218 @@
+# Using Shared Memory with ROS2
+
+This rmw_cyclonedds implementation uses [cyclonedds](https://projects.eclipse.org/projects/iot.cyclonedds) which includes support for fast Shared Memory data transfer based on [iceoryx](https://projects.eclipse.org/projects/technology.iceoryx). Since this feature is still in an experimental stage, it is disabled by default but can be enabled easily in a cyclonedds.xml configuration file.
+
+## Requirements
+
+Currently Shared Memory transport is only supported on Linux. It is available in the rmw_cyclonedds implementation used by the ROS2 Rolling or Galactic Geochelone distribution.
```

Comment:
As a general statement in this document, we prefer "ROS 2" over "ROS2".

Reply:
Fixed throughout the docuement.